src/hotspot/cpu/riscv/stubGenerator_riscv.cpp

```diff
@@ -127,8 +127,20 @@ class StubGenerator: public StubCodeGenerator {
   //     [ return_from_Java     ] <--- sp
   //     [ argument word n      ]
   //      ...
-  // -22 [ argument word 1      ]
-  // -21 [ saved x27            ] <--- sp_after_call
+  // -34 [ argument word 1      ]
+  // -33 [ saved f27            ] <--- sp_after_call
```

Comment:
I think it's better to add some comments about saving FloatRegisters at line 124.